test/utils/test-helpers.js

```diff
@@ -1,20 +1,16 @@
 // In this project, these methods are only used in tests,
 // but plugin tracks also make use of them... so not really extraneous.
-
-import React from 'react';
+import * as React from 'react';
 import ReactDOM from 'react-dom';
-
-// eslint-disable-next-line import/no-extraneous-dependencies
-import { mount } from 'enzyme';
-
-import { requestsInFlight } from '../services';
+import Enzyme from 'enzyme';
 
 import {
-  getTrackObjectFromHGC,
+  requestsInFlight,
   getTrackRenderer,
-} from './get-higlass-components';
-
-import HiGlassComponent from '../HiGlassComponent';
+  getTrackObjectFromHGC,
+  HiGlassComponent,
+// eslint-disable-next-line import/no-unresolved
+} from 'higlass';
```

Comment:
This file will be available to consumers in the final build under `dist/testing.js`. Users should be able to import with. 

During development, Vite aliases `higlass` to `app/scripts/hglib.jsx` but in the final build consumers will get `dist/hglib.js`.